bedrock/firefox/redirects.py

```diff
@@ -653,4 +653,8 @@ def firefox_channel(*args, **kwargs):
     # issue 8641
     redirect(r'^/firefox/windows-64-bit/?$', 'firefox.browsers.windows-64-bit'),
     redirect(r'^/firefox/best-browser/?$', 'firefox.browsers.best-browser'),
+
+    #unfck campaign
+    redirect(r'^firefox/unfuck(/.*)?', '/firefox/unfck'),
```

Comment:
I think the regex can be simplified a bit here, and we can also use the page name for the destination?
```suggestion
    redirect(r'^firefox/unfuck/?$', 'firefox.campaign.unfck.index'),
```

Not a blocker, but having a couple of redirect tests would be good here too for an extra measure.

Reply:
I added one additional redirect (plus tests) in https://github.com/mozilla/bedrock/pull/9454 if it makes things simpler (I'm also happy to rebase if needed, whatever works).